src/stan/io/deserializer.hpp

```diff
@@ -486,237 +436,6 @@ class deserializer {
     }
   }
 
-  /**
-   * Specialization of offset multiplier for `std::vector` with
-   * non-`std::vector` offset and multiplier.
-   *
-   * <p>See <code>stan::math::offset_multiplier_constrain(T, double,
-   * double)</code>.
-   *
-   * @tparam Ret The type to return.
-   * @tparam Jacobian Whether to increment the log of the absolute Jacobian
-   * determinant of the transform.
-   * @tparam Offset Type of offset.
-   * @tparam Mult Type of multiplier.
-   * @tparam LP Type of log probability.
-   * @tparam Sizes A parameter pack of integral types.
-   * @param offset Offset.
-   * @param multiplier Multiplier.
-   * @param lp Reference to log probability variable to increment.
-   * @param vecsize The size of the return vector.
-   * @param sizes Pack of integrals to use to construct the return's type.
-   * bounds.
-   */
-  template <typename Ret, bool Jacobian, typename Offset, typename Mult,
-            typename LP, typename... Sizes,
-            require_std_vector_t<Ret>* = nullptr>
-  inline auto read_offset_multiplier(const Offset& offset,
-                                     const Mult& multiplier, LP& lp,
-                                     const size_t vecsize, Sizes... sizes) {
```

Comment:
Why did this one get deleted?

Reply:
Math handles all the `std::vector` things. Same as with lb/ub/lub constrain

Reply:
Oh duh because we have them in the math library